resources/metadata/whitesourceExecuteScan.yaml

```diff
@@ -519,6 +519,21 @@ spec:
                 type: int
               - name: policy_violations
                 type: int
+      - name: reports
+        type: reports
+        params:
+          - filePattern: "whitesource/**"
+            type: whitesource-ip
+          - filePattern: "whitesource-riskReport.pdf"
+            type: whitesource-ip
+          - filePattern: "**/toolrun_whitesource_*.json"
+            type: whitesource-ip
+          - filePattern: "whitesource/**"
```

Comment:
Here we should catch `piper_whitesource_vulnerability_report.html` which is the vulnerability report.

Reply:
Do we need to upload all reports from whitesource folder (pattern "whitesource/**") or replace it with more specific patterns according to comments?

Reply:
My preference would be replacing them with more specific patterns. @OliverNocon what's your take on that?